robot/Cumulus/resources/NPSPSettingsPageObject.py

```diff
@@ -231,3 +235,14 @@ def check_rd2_is_enabled(self):
                 enabled = True
         return enabled
 
+    @capture_screenshot_on_error
+    def verify_status_to_state_mappings(self, **kwargs):
+        """verifies the default status to state mappings for recurring donations"""
+        for key,value in kwargs.items():
+            locator=npsp_lex_locators['npsp_settings']['erd_state_status_mapping'].format(key)
+            self.selenium.wait_until_page_contains_element(locator, error=f"Couldn't find {key} on the page")
+            if self.npsp.check_if_element_exists(locator):
+                print(f"element exists {locator}")
+                actual_value=self.selenium.get_webelement(locator).text
+                print(f"actual mapping found for {key} is {actual_value}")
```

Comment:
was these print statements added just for troubleshooting or do u intent to keep it?

Reply:
i intend to keep the print statement for better debugging